CryptoTracker/src/components/Currency/index.tsx

```diff
@@ -1,29 +1,51 @@
-import {View} from 'react-native';
-import React from 'react';
+import {Alert, View} from 'react-native';
+import React, {useState} from 'react';
 import {
   Container,
   CryptoIcon,
   IconContainer,
-  SubTitle,
-  Title,
+  TextName,
   Percentage,
 } from './styles';
 import {Cryptocurrency} from '../../interfaces/Cryptocurrency';
+import {SubTitle} from '../../utils/globalStyles';
+import {useCurrency} from '../../store/hooks/useCurrency';
 
-const Currency = ({item}: {item: Cryptocurrency}) => (
-  <Container>
-    <IconContainer>
-      <CryptoIcon source={item.imageUrl} />
+const Currency = ({item}: {item: Cryptocurrency}) => {
+  const [modal, setModal] = useState(true);
+  const {removeCurrency} = useCurrency();
+  const Modal = () => {
+    Alert.alert(
+      `Do you want to delete "${item.symbol}"?`,
+      'A deleted coin cannot be recovered',
+      [
+        {text: 'No', style: 'cancel'},
+        {
+          text: 'Delete',
+          onPress: () => {
+            removeCurrency(item.id);
+            setModal(!modal);
+          },
+        },
+      ],
+    );
+  };
+
+  return (
+    <Container>
+      <IconContainer onPress={() => Modal()}>
```

Comment:
Change name of Modal for showAlert or similar, 
also onPress={showAlert} should work fine